app/build.gradle

```diff
@@ -88,9 +88,10 @@ dependencies {
   compile 'com.android.support:cardview-v7:25.3.1'
   compile 'com.android.support:recyclerview-v7:25.3.1'
 
-  // Google cloud messaging (GCM)
-  compile 'com.google.android.gms:play-services-gcm:9.6.1'
-
+  // Google Play Services availability check
+  compile ("com.google.android.gms:play-services-base:9.6.1")
```

Comment:
Can't fetch required dependencies. IMO there should be `classpath 'com.google.gms:google-services:3.1.0'
` added to the root gradle file